styleguide-app/Examples/Tooltip.elm

```diff
@@ -206,9 +197,19 @@ controlContent =
         ]
 
 
-controlTail : Control Bool
+controlTail : Control (Tooltip.Attribute Never)
 controlTail =
-    Control.bool False
+    Control.map
+        (\bool ->
+            if True then
+                Tooltip.withoutTail
+
+            else
+                -- TODO: change `withoutTail` to take
+                -- a bool or expose a `withTail` from Tooltip.
+                Tooltip.css []
+        )
+        (Control.bool False)
```

Comment:
Not sure I follow this part... Why do we need to pass in `Never`?  And Why was it hard-coded to `Control.bool False` beforehand? 🤔 

Testing out the styleguide example tooltip, I can no longer toggle the "tail" by checking `withoutTail`. (But on the master branch, toggling it does work.)

Reply:
OOPS I see what I did 🙈 `if True then` Thanks for noticing!!!

---

As to how this is _supposed_ to work when I don't hardcode bools:

`Control.bool` takes a bool argument so that it can create a checkbox that's initialized in the state you want.

So, if I wanted the tooltip example to show the without-tail behavior by default, I would change `Control.bool False` to `Control.bool True`

Reply:
Oh there's a confounding problem -- `withAlignment` in the Tooltip component wipes the no-tail setting. That's not good! Alignment and tail-less-ness should be independent properties (you can be an evil hamster OR an evil mouse, after all!)

I'm not going to mess with the internals of tooltip in this PR. But definitely good to know!